app/models/concerns/currencies.rb

```diff
@@ -62,6 +62,7 @@ module Currencies
     ILS: 'Israeli New Sheqel',
     INR: 'Indian Rupee',
     ISK: 'Icelandic Króna',
+    IRR: 'Iranian Rial',
```

Comment:
If you don't mind, could you move it right before the `ISK` so that it keeps the alphabetic order on the currency codes?